db/migrate/20151115221849_add_subscribed_to_participant_activity_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddSubscribedToParticipantActivityToUsers < ActiveRecord::Migration
+  def up
+    add_column :users, :subscribed_to_participant_activity, :boolean, default: false
+    User.update_all(subscribed_to_participant_activity: false)
```

Comment:
I think you should add a validation to the `User` model for these fields ... as in just check they're presrent